app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -307,14 +307,14 @@ def alert_build_edit_screen
       if @alert.responds_to_events == "_hourly_timer_"                    # Check for hourly timer event
         @edit[:new][:exp_event] = @alert.responds_to_events
       else
-        exp_event = MiqEventDefinition.find_by_name(@alert.responds_to_events)
+        exp_event = MiqEventDefinition.find_by(:name => @alert.responds_to_events)
```

Comment:
Maybe make this a helper method to clean up the rest of the code climate complaints:

```ruby
# method name is a WIP
def fetch_exp_event
  MiqEventDefinition.find_by(:name => @alert.responds_to_events)
end
```

Might make things in this controller more confusing, so not sure...

Reply:
@NickLaMuro that may be more refactoring for another PR.  This PR is laser-focused on converting a single deprecation.

Reply:
@chessbyte sounds good.  I was reluctant to make the above comment as I was thinking a similar thing.